src/components/bookmark/svg.js

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { UncontrolledTooltip } from "reactstrap";
+import PropTypes from "prop-types";
+
+const SVG = (
+  { ...props },
+  {
+    style = {},
+    fill = "#000000c2",
+    width = "40px",
+    className = "svgIcon",
+    height = "40px",
+    viewBox = "0 0 32 32"
+  }
+) => (
+  <div>
+    <span onClick={props.handlebookmark} id="bookmark">
+      <svg
+        width={width}
+        style={style}
+        height={height}
+        viewBox={viewBox}
+        className={className}
+      >
+        <path
+          d="M19.385 4h-9.77A2.623 2.623 0 0 0 7 6.615V23.01a1.022 1.022 0 0 0 1.595.847l5.905-4.004 5.905 4.004A1.022 1.022 0 0 0 22 23.011V6.62A2.625 2.625 0 0 0 19.385 4zM21 23l-5.91-3.955-.148-.107a.751.751 0 0 0-.884 0l-.147.107L8 23V6.615C8 5.725 8.725 5 9.615 5h9.77C20.275 5 21 5.725 21 6.615V23z"
```

Comment:
Declare this svg path in a different file and import it here rather than writing the whole path in the component

Reply:
It has been changed